job-server/test/spark.jobserver/JobSpecBase.scala

```diff
@@ -28,7 +28,8 @@ trait JobSpecConfig {
       "spark.jobserver.max-jobs-per-context" -> MaxJobsPerContext,
       "akka.log-dead-letters" -> 0,
       "spark.master" -> "local[4]",
-      "context-factory" -> contextFactory
+      "context-factory" -> contextFactory,
+      "akka.test.single-expect-default" -> 6000
```

Comment:
nope, you're right but I saw a couple of test failing on a slow machine that I have with 2.11